Lib/test/test_embed.py

```diff
@@ -1303,6 +1303,48 @@ def test_init_setpythonhome(self):
         self.check_all_configs("test_init_setpythonhome", config,
                                api=API_COMPAT, env=env)
 
+    @unittest.skipUnless(MS_WINDOWS, 'Windows only')
```

Comment:
This change applies to all platforms, right? Even though we only saw the original issue on Windows.